### DISCUSSION TITLE
Can I join?

### DISCUSSION THREAD

**Discussion by TheBugCoder** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#197] [Q&A] [answered]
Can I join? I can HTML, CSS, and JavaScript. This is my Scratch profile: https://scratch.mit.edu/users/TheBugCoder/

**Comment by Shluffy** [comment]
Make a bookmarklet and we will see

**Reply by TheBugCoder** [reply]
For Scratch?

**Comment by TheBugCoder** [comment]
Open in TurboWarp
javascript: url = window.location.href; window.open("https://turbowarp.org/" + url.substring(33));

**Reply by Shluffy** [reply]
Because this already exists, would it be possible to make a different one as this is a little different but exists? maybe try something that switches the site to dark mode, or something simple. check the `/bookmarklets/` directory to make sure it doesn't exist

**Reply by TheBugCoder** [reply]
Ok! :)

**Reply by TheBugCoder** [reply]
Sorry for the late response, I was on holidays! I can't really think of a bookmarklet, but I can make the website better.

**Reply by imlens** [reply]
How about dark mode? Be creative! Do you have any problems in the SE? Make a bookmarklet that will solve that problem
@TheBugCoder

**Reply by TheBugCoder** [reply]
> How about dark mode? Be creative! Do you have any problems in the SE? Make a bookmarklet that will solve that problem
> @TheBugCoder

Thanks! I will try!

**Reply by TheBugCoder** [reply]
> Because this already exists, would it be possible to make a different one as this is a little different but exists? maybe try something that switches the site to dark mode, or something simple. check the `/bookmarklets/` directory to make sure it doesn't exist

javascript:document.getElementsByClassName("see-inside-button")[0].click();void(0); See inside a project.

**Reply by Shluffy** [reply]
> > Because this already exists, would it be possible to make a different one as this is a little different but exists? maybe try something that switches the site to dark mode, or something simple. check the `/bookmarklets/` directory to make sure it doesn't exist
> 
> javascript:document.getElementsByClassName("see-inside-button")[0].click();void(0); See inside a project.

you are in! let me invite you

**Reply by TheBugCoder** [reply]
Yes! Thanks! :D

**Comment by Shluffy** [comment]
Switched to Q&A

**Comment by Shluffy** [comment, marked as answer]
Above threads answer this
